indra/newview/pipeline.cpp

```diff
@@ -7880,7 +7880,6 @@ void LLPipeline::renderFinalize()
 
     if (RenderBufferVisualization > -1)
     {
-        finalBuffer = &mPostMap;
```

Comment:
So is this correct that you overwrite the buffer you have been written to last? Or am I missing something in visualizeBuffers()?

Also could remove the unused `aa_type` LLCachedControl.

Reply:
Yes that's correct. `visualizeBuffers()` overwrites the output buffer with whatever buffer is being requested to be visualized.